data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -158,6 +163,11 @@ backend {{ back }}
 {%         endif %}
 {%         if back_config.mode is vyos_defined %}
     mode {{ back_config.mode }}
+{%             if back_config.mode == 'http' and back_config.http_response_headers is vyos_defined %}
```

Comment:
I wonder if we can/should add a `verify()` code path to only allow `back_config.http_response_headers` to be set if  `back_config.mode == 'http'`?

Reply:
Probably a good idea, I have added this validation.